examples/qml_features/rust/src/nested_qobjects.rs

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+#[cxx_qt::bridge(cxx_file_stem = "nested_qobjects")]
+mod ffi {
+    unsafe extern "C++" {
+        #[cxx_name = "InnerObject"]
```

Comment:
seems we need to
- read all `extern "C++"` blocks
- create a map of `T -> S` where `#[cxx_name = "S"] type T ...`
- when writing the cpp types for this module, if we hit `T` write `S`

Reply:
Done in a commit, do we also need to consider any namespace :thinking: